app/components/SearchTableLayout.tsx

```diff
@@ -4,13 +4,21 @@ import SortableTableHeader from 'components/SortableTableHeader';
 import SearchBox from 'components/SearchBox';
 interface Props {
   handleEvent: (...args: any[]) => void;
+  handleSelectAll?: (...args: any[]) => void;
```

Comment:
I know there's a lot of examples of this pattern `..args: any[]` all over, but we're trying to avoid being so general going forward, and I imagine at some point we'll end up doing some tech debt to fix these. I would suggest
```suggestion
  handleSelectAll?: (boolean) => void;
```

Reply:
Actually, this is the only change I see needed in this PR & it's pretty minor. Lets just merge it and I'll add this change in my current PR

Reply:
I do usually try to make them more specific, but this was probably left over from an interim step when I was figuring things out and forgot to update it :)